README.md

```diff
@@ -4,227 +4,211 @@
-the "Running" section.
-
-The first thing you will need is cloning the project:
-> `$ git clone git@github.com:3scale/apisonator.git`
+## Description
 
-Next cd into the directory, `cd apisonator`.
-
-### Containerized environment
-
-#### Prerequisites
-
-* Docker (requires version 1.10.0 or later)
-
-#### With Docker
-
-This requires GNU Make and has a single step:
-
-1. Run: `make dev`
-
-This command will take care of downloading and building all dependencies. Once
-that is done, the process will be way faster the next time.
-
-The project's source code will be available in `~/apisonator` and sync'ed with
-your local apisonator directory, so you can edit files in your preferred
-environment and still be able to run whatever you need inside the Docker
-container.
+This is Red Hat 3scale API Management Platform's Backend.
 
-The listener service port (3000 by default) is automatically forwarded to the host machine. The `make dev` command can be executed with the environment variable `PORT` set to
-a different desired listener port value to forward on the host machine.
+It has the following components:
 
-This Docker container is persistent, so your changes will be kept the next time
-you enter it.
+### Apisonator listener
 
-Getting rid of the persistent container is done with `make dev-clean`, whereas
-removing its image is done using `make dev-clean-image`.
+It provides the point of entry of the API Management Platform's Backend.
+The Service management API (SM API) is provided to authorize and report consumer
+API requests.
 
-Alternatively you can start a container with the service running with
-`make dev-service`. This rule is intended for when you want to test
-the service endpoint.
+Three main operations can be performed with this API:
 
-#### Maintain your dependencies up-to-date
+ * Report: Reports a number of hits to one or more metrics
+ * Authorize: Authorize a request
+ * Authrep: Combination of both the above
 
-Changes in code sometimes translate into changes in dependencies. If that is the
-case, your container will lag behind in dependencies, and some things might just
-start breaking. You might want to make a habit of making sure dependencies are
-updated when you enter your container. Run this from the project directory:
+Make sure to read the corresponding Swagger-generated documentation of this
+operations, located in `docs/active_docs/Service Management API.json`
 
-> $ `bundle install`
+It attempts to respond with lowest possible latency, performing minimal work
+and offloading other work to Apisonator workers by enqueuing tasks into job queues.
 
-The container image has additional tools to handle these dependencies for
-multiple Ruby versions. Check out the `scripts` directory if you are curious.
+This component needs access to a Redis database to perform the following actions:
+ * Enqueue reports, which will be processed by the Apisonator worker component
+ * Perform authorization of the requests
```

Comment:
I would add a brief explanation about what *authorization* means. Something like:

```
Authorization task is based several validations. Two main validations are:
- *api key* is valid
- usage metrics are within limits
```

Feel free to change the explanation to something else for clearness


Maybe getting too much into detail. Up to you.

Reply:
:+1: